packages/@aws-cdk/aws-appconfig/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@aws-cdk/aws-appconfig",
+  "version": "1.23.0",
+  "description": "The CDK Construct Library for AWS::AppConfig",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "jsii": {
+    "outdir": "dist",
+    "targets": {
+      "dotnet": {
+        "namespace": "Amazon.CDK.AWS.AppConfig",
+        "packageId": "Amazon.CDK.AWS.AppConfig",
+        "signAssembly": true,
+        "assemblyOriginatorKeyFile": "../../key.snk",
+        "iconUrl": "https://raw.githubusercontent.com/aws/aws-cdk/master/logo/default-256-dark.png"
+      },
+      "java": {
+        "package": "software.amazon.awscdk.services.appconfig",
+        "maven": {
+          "groupId": "software.amazon.awscdk",
+          "artifactId": "appconfig"
+        }
+      },
+      "python": {
+        "distName": "aws-cdk.aws-appconfig",
+        "module": "aws_cdk.aws_appconfig"
+      }
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aws/aws-cdk.git",
+    "directory": "packages/@aws-cdk/aws-appconfig"
+  },
+  "homepage": "https://github.com/aws/aws-cdk",
+  "scripts": {
+    "build": "cdk-build",
+    "watch": "cdk-watch",
+    "lint": "cdk-lint",
+    "test": "cdk-test",
+    "integ": "cdk-integ",
+    "pkglint": "pkglint -f",
+    "package": "cdk-package",
+    "awslint": "cdk-awslint",
+    "cfn2ts": "cfn2ts",
+    "build+test+package": "npm run build+test && npm run package",
+    "build+test": "npm run build && npm test",
+    "compat": "cdk-compat"
+  },
+  "cdk-build": {
+    "cloudformation": "AWS::AppConfig"
+  },
+  "keywords": [
+    "aws",
+    "cdk",
+    "constructs",
+    "AWS::AppConfig",
+    "aws-appconfig"
+  ],
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "jest": {
+    "moduleFileExtensions": [
+      "js"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 60,
+        "statements": 80
+      }
+    }
```

Comment:
It seems I missed to fix up the template that generates this. Can you point me at it?

Jest config [is now centralized](https://github.com/aws/aws-cdk/commit/2c5c88d33d51746eb35fe46068b0c5fd509d8d7a), so all is needed here is `"jest": {}` unless you want to override.

Reply:
It's there:

https://github.com/aws/aws-cdk/blob/491e2d970c849cbc00e7cb3def927b12054d572f/packages/%40aws-cdk/cfnspec/build-tools/create-missing-libraries.ts#L170-L180

Reply:
Thanks!

Reply:
Did it here.